src/llv8.h

```diff
@@ -476,6 +482,7 @@ class LLV8 {
   friend class JSRegExp;
   friend class JSDate;
   friend class CodeMap;
+  friend class llnode::FindJSObjectsVisitor;
```

Comment:
I moved GetTypeName to be part of LLV8 which saves pre-declaring a messy function definition that's not part of any class in llv8.h but FindJSObjectsVisitor::IsAHistogramType still needs to look at the types to determine if we are interested in a given item on the heap.
I'm not sure I can sensibly move that function to llv8.h, it belongs with the visitor class, but I'm open to suggestions!